src/main/java/frc/robot/autos/exampleAuto.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.autos;
+
+import frc.robot.Constants;
+import frc.robot.subsystems.Swerve;
+
+import java.util.List;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.math.trajectory.TrajectoryGenerator;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.SwerveControllerCommand;
+
+public class exampleAuto extends SequentialCommandGroup {
```

Comment:
I think this was copied with the other code. Is this the example we want for our Auto's? It appears quite different from what we've done before?

Reply:
This was an example that we do not need. Removed.